core/domain/stats_domain_test.py

```diff
@@ -70,9 +76,11 @@ def setUp(self):
         self.exploration_stats = self._get_exploration_stats_from_dict(
             self.exploration_stats_dict)
 
-    def _get_exploration_stats_from_dict(self, exploration_stats_dict):
-        """Converts and returns the ExplorationStats object from the given
-        exploration stats dict.
+    def _get_exploration_stats_from_dict(
+        self, exploration_stats_dict:
+        stats_domain.ExplorationStatsDict) -> stats_domain.ExplorationStats:
+        """Converts and returns the ExplorationStats
+        object from the given exploration stats dict.
```

Comment:
The line above this looks too short -- can you use the full 80-char limit?

Reply:
Done